packages/gatsby-codemods/src/transforms/global-graphql-calls.js

```diff
@@ -119,7 +123,12 @@ module.exports = (file, api, options) => {
   let tag = tags.get(0)
 
   const useImportSyntax =
-    root.find(j.ImportDeclaration, { importKind: `value` }).length > 0
+    root.find(j.ImportDeclaration, { importKind: `value` }).length > 0 ||
+    root.find(j.VariableDeclarator, {
```

Comment:
This is an assumption. If there isn't _any_ import or require statement, we have to make a decision here. I think we should prefer esm, but I'd rather not make any assumption--thus see above where I _think_ we'll always have at least one import/require statement in the file